.github/workflows/release-publish.yml

```
@@ -0,0 +1,48 @@
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+
+      - name: Gradle publish
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2215:warning:2:1: This flag is used as a command name. Bad line break or missing [ .. ]?

(shellcheck)